creator-node/src/services/prometheusMonitoring/prometheusRegistry.js

```diff
@@ -0,0 +1,43 @@
+const promClient = require('prom-client')
+
+module.exports = class PrometheusRegistry {
+  init({ collectDefaultMetrics = true }) {
+    this.register = new promClient.Registry()
+
+    // Add a default label which is added to all metrics
+    this.register.setDefaultLabels({
+      serviceType: 'audius_content_node'
+    })
+
+    this.setupDefaultMetrics(collectDefaultMetrics)
+  }
+
+  getAllMetricData() {
+    return this.register.metrics()
+  }
+
+  async getMetricInstance(name) {
+    return this.register.getSingleMetric(name)
+  }
+
+  setupDefaultMetrics(collectDefaultMetrics) {
+    if (collectDefaultMetrics) {
+      promClient.collectDefaultMetrics({
+        register: this.register
+      })
+    }
+
+    this._createHTTPRequestTimerMetric()
```

Comment:
a better approach would probably be to have every prometheus metric defined in a separate file and register them all here - wdyt

Reply:
i don't feel too strongly about this topic. i do agree declaring in one place is easier than two. especially if we could have something like 
```js
createOrGetMetricInstance(metricName, metricType) {
  if (this.registeredMetrics[metricName]) {
    return this.registeredMetrics[metricName]
  }

  const metric = new promClient[metricType]({
    name: metricName,
    help: metricName
  }
  this.registeredMetrics[metricName] = metric
  return metric
}
```

Reply:
Indeed, we have the same sort of [error](https://github.com/prometheus/client_python/blob/748ffb00600dc25fbd22d37d549578e8e370d996/prometheus_client/registry.py#L44) on the Python side which is why I'm doing a similar dance, but in a [`defaultdict` style](https://github.com/AudiusProject/audius-protocol/blob/8a07f5d9482f047742d2b2dd262fa6b775ae76d0/discovery-provider/src/utils/prometheus_metric.py#L20-L27).

I looked through the docs for this bit which I was hoping would help address my ethos argument (not that I'm still pursuing that ethos after your constants.js changes! 🔥 ), but it backfired on me! 😅 

* https://prometheus.io/docs/practices/instrumentation/#avoid-missing-metrics

I thought I remembered a section regarding metric robustness over faulty networks, but I may have just confused that ethos with [MQTT's](https://en.wikipedia.org/wiki/MQTT):

> It is designed for connections with remote locations that have devices with resource constraints or limited network [bandwidth](https://en.wikipedia.org/wiki/Bandwidth_(computing)).

In either case, you can resolve this thread since I'm in line with your constant.js changes/benefits!

Reply:
resolving per discussion